ompi/mca/io/romio314/romio/configure.ac

```diff
@@ -1360,11 +1360,53 @@ if test "$pac_cv_have_statfs" = yes ; then
     AC_DEFINE(HAVE_STRUCT_STATFS,1,[Define if struct statfs can be compiled])
 fi
 
+AC_MSG_CHECKING([for f_type member of statfs structure])
+AC_TRY_COMPILE([
```

Comment:
You should use `AC_CHECK_MEMBERS` instead of `AC_TRY_COMPILE` (`AC_TRY_COMPILE` is deprecated, anyway).